_templates/base.html

```diff
@@ -75,7 +75,7 @@
     <script src="{% static '/static/js/jquery.cookie.js' %}"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js" integrity="sha256-Sk3nkD6mLTMOF0EOpNtsIry+s1CsaqQC1rVLTAy+0yc= sha512-K1qjQ+NcF2TYO/eI3M6v8EiNYZfA95pQumfvcVrTHtwQVDG+aHRqLi/ETn2uB+1JqwYqVG3LIvdm9lj6imS/pQ==" crossorigin="anonymous"></script>    {% block extra_js %}
     {% endblock %}
-    <script async src="{% static '/static/js/main.js' %}"></script>
+    <script async src="{% static '/static/js/dl.js' %}"></script>
```

Comment:
OK @jodok firstly this means constructing the button slightly differently as the previous code used JS to redirect to the download link, that's fine. But using the code above results in "Uncaught ReferenceError: analytics is not defined"

I'm just using an onclick event on a link at the moment, but to be honest I'm not really sure how we're tracking things anymore. If the old approach is no longer correct, it may be easier for you to finish the JS side of this :)